testng-core/src/test/kotlin/test/hook/samples/ConfigurableFailureSample.kt

```diff
@@ -0,0 +1,15 @@
+package test.hook.samples
+
+import org.testng.IConfigureCallBack
+import org.testng.ITestResult
+import org.testng.annotations.Test
+import test.hook.HOOK_INVOKED_ATTRIBUTE
+
+class ConfigurableFailureSample:BaseConfigurableSample() {
```

Comment:
fix style

Reply:
Style? I have run the autoStyle plugin and then only raised the PR. Can you please let me know what style are you referring to ?